src/server.js

```diff
@@ -53,9 +54,14 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
+		if (config.https.ca.length && !fs.existsSync(caPath)) {
```

Comment:
Personally, its easier to understand what is `config.https.ca` then `Helper.expandHome(config.https.ca)`. Speciel when same style is applied on `config.https.key` and `config.https.certificate`